wiki/Contests/VMC/1/en.md

```diff
@@ -0,0 +1,125 @@
+- **Submissions must use one of the [selected songs](#songs)**
+- **Submissions must be in `.osz` format.**
+- **Beatmaps should abide by the [ranking criteria](/wiki/Ranking_Criteria).** While we can be forgiving for mistakes, submissions that could not be ranked without major changes will be disqualified.
+- **Submissions must not be publicly revealed until after the results are announced.**
+- **Excluding metadata, submissions must not include anything that could reveal your identi.** Metadata will be anonymized before submissions are relayed to judges.
+- **Collaborations are not allowed**. This is a solo contest.
+- **Submissions must not include storyboards or videos.**
+- **Do not edit any settings in the .osu that would not be possible through the editor itself.** You are permitted to change any difficulty settings, backgrounds, green lines, SV, hitsounds, etc.
+- **You must submit only one difficulty.** Only one entry is allowed.
+
+## Judging Criteria
+| Position | Points | Description
+| :-- | :-- | :-- |
+| Relevance | 30 |  How well does the map fit the song? Do the elements of the map match the elements of the song, and is the map fundamentally inseparable from the song it is paired with?
+| Creativity | 25 | How unique and outstanding is the map and the elements within it compared to the current mapping meta? How well does it stand out from the crowd? Did you make the judges say "wow!", "that's cool!", or "that's super interesting!"? Generic mapping that just copies the current mapping meta or concepts from other "famous" maps is discouraged.
+| Expertise | 20 | This refers to how well the map executes the elements it presents. This encompasses the technical skills of the mapper, and also the consistency, structure, and style of the map. Judges will score this entry considering the overarching picture of the map, and will avoid nitpicking small details.
+| Overall Impression | 20 | This contest is judged by some of the most prominent vocaloid mappers in the community, and you better bet that all of them have a personal bias with regard to how vocaloid songs are mapped. Judges will give comments regarding their scoring.
+| Hitsounding | 5 | Rankable hitsounds can get up to 2 points, but hitsounds that fundamentally add to the experience and compliment the map in a unique way can get the full 5 points.
+
+## Songs
+
+Five (5) songs were selected from the Featured Artist library. These songs were selected based on several factors, including whether they had been ranked in the past, as well as providing a variety of genres and artists to choose from.
+
+### MIMI - Moment (3:34 / 126 BPM)
+
+[Download the beatmap template here!](https://assets.ppy.sh/artists/77/osz/MIMI%20-%20Moment.osz)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/77/osz/MIMI%20-%20Moment.mp3" type="audio/mpeg">
+</audio>
```

Comment:
you'll have to delete these previews because html is not supported in the wiki. please convert this whole section into a table